README.md

```diff
@@ -57,6 +57,37 @@ as |slot|}}
       {{infoBar.summary}}
     </div>
   {{/block-slot}}
+  {{#block-slot slot 'view-controls' as |viewControl viewLevel onDetailChange|}}
+    <div class="button-bar {{ viewControl.detailLevel }}">
+    {{#if viewLevel.low}}
+      {{frost-button
+        disabled=(eq viewControl.detailLevel 'low')
+        onClick=(action onDetailChange 'low')
+        priority="tertiary"
+        size="small"
```

Comment:
Can we use ' instead of " ?  We have been using single quotes all over the place.  There are lots of occurrences.